BUILD.md

```diff
@@ -98,31 +100,29 @@ Download and install the standalone [Windows 10 SDK](https://dev.windows.com/en-
 
 * `C:\Program Files (x86)\Windows Kits\10\bin\x86`
 * `C:\Program Files (x86)\Windows Kits\10\Redist\ucrt\DLLs\x86`
-* `C:\Users\user\AppData\Local\Programs\Python\Python35-32\Lib\site-packages\PyQt5\Qt\bin`
+* `C:\Users\user\AppData\Local\Programs\Python\Python36-32\Lib\site-packages\PyQt5\Qt\bin`
 
 If you want to build the installer:
 
-* Go to http://nsis.sourceforge.net/Download and download the latest NSIS. I downloaded `nsis-3.01-setup.exe`.
+* Go to http://nsis.sourceforge.net/Download and download the latest NSIS. I downloaded `nsis-3.02.1-setup.exe`.
 * Add `C:\Program Files (x86)\NSIS` to the path.
 
 If you want to sign binaries with Authenticode:
 
-* You'll need a code signing certificate. I roughly followed [this guide](http://blog.assarbad.net/20110513/startssl-code-signing-certificate/) to make one using my StartSSL account.
+* You'll need a code signing certificate. I got an open source code signing certificate from [Certum](https://www.certum.eu/certum/cert,offer_en_open_source_cs.xml).
 * Once you get a code signing key and certificate and covert it to a pfx file, import it into your certificate store.
 
 ### To make a .exe:
 
-For PyInstaller to work, you might need to edit `Scripts\pyinstaller-script.py` in your Python 3.5 folder, to work around [this bug](https://stackoverflow.com/questions/31808180/installing-pyinstaller-via-pip-leads-to-failed-to-create-process) in pip.
-
 * Open a command prompt, cd into the onionshare directory, and type: `pyinstaller install\pyinstaller.spec`. `onionshare-gui.exe` and all of their supporting files will get created inside the `dist` folder.
```

Comment:
I didn't need the trailing dot at the end of 'install\pyinstaller.spec.' - not sure if it's a big deal?

Reply:
Oh the dot is just a period at the end of the sentence, it isn't part of the code block :)

Reply:
@micahflee ah my bad :) I thought onionshare-gui.exe was a sort of 'output' argument for the pyinstaller command.